lib/fog/ovirt/core.rb

```diff
@@ -10,6 +10,7 @@ module Errors
       class ServiceError < Fog::Errors::Error; end
       class SecurityError < ServiceError; end
       class NotFound < ServiceError; end
+      class OvirtError < Fog::Errors::Error; end
```

Comment:
Why not splitting errors to fog-generated base and derived wrapper exception that will save the original exception object:

``` ruby
class OvirtError < Fog::Errors::Error; end

class OvirtClientError < OvirtError
  attr_reader :original
  def initialize(client_exception)
    @original = client_exception
    super("Ovirt client returned an error: #{client_exception.message}")
  end
end
```

Reply:
done